sources/web/datalab/polymer/modules/drive-file-manager/drive-file-manager.html

```diff
@@ -13,7 +13,7 @@
 -->
 
 <link rel="import" href="../../modules/utils/utils.html">
-<link rel="import" href="../api-manager-factory/api-manager-factory.html">
+<link rel="import" href="../api-manager/api-manager.html">
```

Comment:
Unused? ApiManager appears only in a comment in drive-file-manager.ts (which probably should be fixed).